### PR TITLE
ADD: MRR data testing examples

### PR DESCRIPTION
**Pull request by rcjackson**
I am adding test MRR2 data for the xradar MRR2 reader.

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@rcjackson Would it work to add the data gzipped? We can easily use fixtures to extract it in the test suite.

File: open_radar_data/registry.txt [hunk 1]
```diff
@@ -46,12 +49,6 @@ T_PAZD63_C_LFPW_20230420065331.h5 7a04d1e642839fbcc12736f39e8d845cd44f56387e7829
 T_PAZD63_C_LFPW_20230420065831.h5 40cbb771bbe718c3fbdae91ab209e1f00759290ce01cee866e8e1c0e788aff91
 T_PAZE63_C_LFPW_20230420065446.h5 bf65fefe5d46530cd5d210485dd8725dc23fe66945c0871041cf7ecbbbe19b86
 T_PAZE63_C_LFPW_20230420065946.h5 c6db05d31a6e0209c1056f61f8839c2bf3eacf4869263d8393643cb05dc15cad
-cfrad.20080604_002217_000_SPOL_v36_SUR.nc 67821b6c2bb0f27b5de49dee636f36e6e5bbad95f1ee168cb2d1af48e98992fe
```
Comment (kmuehlbauer) [inline open_radar_data/registry.txt:49]: This line, and those below, look like they should not be removed.

**Comment by rcjackson** [comment]
I've fixed the registry to include all of the files and gzipped the MRR2 data.

**Review by kmuehlbauer — COMMENTED** [review]
Thanks @rcjackson, LGTM. I'm not sure if we should make a restructure for the README.md. But otherwise this is good to go.

**Review by kmuehlbauer — APPROVED** [review]

**Comment by kmuehlbauer** [comment]
> I'm not sure if we should make a restructure for the README.md.

@mgrover1 The list in the readme becomes longer and longer. It's not easy to see what kind of data is provided. Maybe we should make subsections for the different radar file formats? What do others think?

I think this should be done in a separate PR.

**Comment by mgrover1** [comment]
I think we should add some dynamic documentation here, which could be tracked in a separate issue

**Pull request merged**